backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/organization/impl/OrgDAO.java

```diff
@@ -374,4 +376,20 @@ public String getOrgName(@NonNull final OrgId orgId)
 		return getById(orgId).getName();
 	}
 
+	@NonNull
+	@Override
+	public String getOrgLanguageOrLoggedInUserLanguage(@NonNull final OrgId orgId)
```

Comment:
i think this kind of method shall go to BPartnerOrgBL.
Pls use `getOrgInfoById` to get the org info.
U can use de.metas.bpartner.service.impl.BPartnerBL#getLanguage(de.metas.bpartner.BPartnerId) to get the language.

Reply:
I wanted to avoid 2 potential hits to the DB (one for orgInfo, one for BP language). I just saw the orgInfo cache. moved to BPartnerOrgBL.